ui/src/app/settings/reducers/general/minHweKernels.js

```diff
@@ -0,0 +1,25 @@
+import produce from "immer";
+
+const minHweKernels = produce(
```

Comment:
this reducer should just be called `general`, so please move this file to `general.js` and rename `const minHweGenerals` here to `general`. 

The reason being, all of these reducers for general properties are going to follow the same pattern, so instead of having nineteen distinct reducers, we can just have one with fallthrough cases.

e.g.
```
case "FETCH_GENERAL_MIN_HWE_KERNELS_SUCCESS":
case "FETCH_GENERAL_OSINFO_SUCCESS":
case ...
  draft[`${action.meta.method}`].loading = false;
  draft[`${action.meta.method}`].items = ...;
  ...
````

Currently, you'll have to parse `action.meta.method` and strip `general.` to get the key for the method name (which is also a suitable model name in state, e.g. `state.general.min_hwe_kernels`). I'll be landing a branch soon with a refactoring that will mean `meta.method` no longer has the `general.` prefix, and will be a value like `min_hwe_kernels`.